patches/components-omnibox-browser-omnibox_edit_model.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/omnibox/browser/omnibox_edit_model.cc b/components/omnibox/browser/omnibox_edit_model.cc
+index 3cfedd66eff30f1966c10fdb6a8b62961a61a8bb..23217e918fc42a4c5e51cf34b6e9e5ead7888aa3 100644
+--- a/components/omnibox/browser/omnibox_edit_model.cc
++++ b/components/omnibox/browser/omnibox_edit_model.cc
+@@ -384,6 +384,7 @@ void OmniboxEditModel::AdjustTextForCopy(int sel_min,
+       (*text == display_text_ || *text == url_for_editing_)) {
+     *url_from_text = controller()->GetLocationBarModel()->GetURL();
+     *write_url = true;
++    HANDLE_BRAVE_SCHEME_URL
```

Comment:
this should be `BRAVE_ADJUST_TEXT_FOR_COPY`

Reply:
Done.